codes/plotting/gen_fig4.py

```diff
@@ -426,6 +495,13 @@ def main():
 
     leg_ax.axis("off")
 
+    df_all_countries = pd.concat(dfs_by_country).drop(["date_str"], axis=1)
+
+    if save_data:
+        out_fn = fig_data_dir / fig_data_fn
+        print("saving fig data in {0}".format(out_fn))
+        df_all_countries.to_csv(out_fn, index=False)
```

Comment:
I think we're gonna keep these files in github as well so can you commit the file that gets output here? `results` used to be under gitignore, but the latest version on master has `results/source_data` available for versioning.

Reply:
yep put that in the newest commit. It's worth noting that for many countries the figure ends on 3-18 because that's when the model predictions i have end -- once those get updated this csv will change